elements/coupling-view/couple-element.html

```diff
@@ -0,0 +1,153 @@
+                    </div>
+                </paper-item>
+           </div>
+            <div class="card-actions layout horizontal end-justified">
+                <paper-button on-tap="openDialog" raised>edit</paper-button>
+            </div>
+        </paper-card>
+
+        <paper-dialog id="remoteCameraDialog" with-backdrop>
+            <h2>Set remote camera</h2>
+            <paper-listbox id="listbox" attr-for-selected="camera" selected="{{_getSelected(camera, remoteCameras)}}">
+                <template is="dom-repeat" items="{{remoteCameras}}">
+                    <paper-item camera="{{item}}">
+                        {{item.address}}
+                    </paper-item>
+                </template>
+            </paper-listbox>
+            <div class="buttons">
+                <paper-button dialog-dismiss>Cancel</paper-button>
+                <paper-button on-tap="saveRemoteCamera">Save</paper-button>
+            </div>
+        </paper-dialog>
+
+   </template>
+
+    <script>
+        (function () {
+            class coupleElement {
+                beforeRegister() {
+                    this.is = "couple-element";
+
+                    this.properties = {
+                        camera: Object,
+                        remoteCameras: {
+                            type: Object,
+                            value: function() {
+                                return [];
+                            }
+                        }
+                    };
+                }
+
+                _getRemoteCameraAddress(id, remoteCameras) {
+                    if (id < 0) {
+                        return "Camera not yet coupled!";
+                    }
+                    const remoteCamera = remoteCameras.find((camera) => camera.id === id);
+                    if (remoteCamera) {
+                        return remoteCamera.address;
+                    }
+                    return "Camera not found!";
+                }
+
+                _getSelected(camera, remoteCameras) {
+                    if (!camera || camera.remoteCameraId === -1) {
+                        return undefined;
+                    }
+                    return remoteCameras.find((remoteCamera) => remoteCamera.id === camera.remoteCameraId);
+                }
+                
+                saveRemoteCamera() {
+                    this.set("camera.remoteCameraId", this.$.listbox.selected.id);
+                    this.$.ajaxSetRemoteCamera.body = {
+                        remoteCameraId: this.camera.remoteCameraId
+                    };
+                    this.$.ajaxSetRemoteCamera.generateRequest();
+                }
+                
+                onSetRemoteCamera(event) {
+                    if (event.detail.response.success) {
+                        this.$.remoteCameraDialog.close();
+                    } else {
+                        console.log(event.detail.response);
```

Comment:
I added this in case something weird happens. Wouldn't expect it though. It is only for the presentation.